backend/src/main/scala/com/softwaremill/bootzooka/user/UserApi.scala

```diff
@@ -44,36 +44,36 @@ class UserApi(http: Http, auth: Auth[ApiKey], userService: UserService, xa: Tran
     }
 
   private val changePasswordEndpoint = secureEndpoint.post
-    .in(UserPath / "changepassword")
+    .securityIn(UserPath / "changepassword")
     .in(jsonBody[ChangePassword_IN])
     .out(jsonBody[ChangePassword_OUT])
-    .serverLogic { case (authData, data) =>
+    .serverSecurityLogic(authData => auth.applyEither(authData).map(_.left.map(exceptionToErrorOut)))
```

Comment:
can't we use `.toOut` here as for server logic?

Reply:
Not if I understand it correctly here we just mapping left while `.toOut` perform mapping on both left and right

Reply:
I didn't understand correctly fixed